<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>552</width>
    <height>432</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>380</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>11</y>
     <width>501</width>
     <height>361</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="layout_projection">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Projection</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_projection">
        <item>
         <property name="text">
          <string>central</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>parallel</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="layout_backgroundColor">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Background color</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_backgroundColor">
        <item>
         <property name="text">
          <string>white</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>black</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>red</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>blue</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>green</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Edges</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="layout_edgesColor">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Color</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_edgesColor">
        <item>
         <property name="text">
          <string>white</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>black</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>red</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>blue</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>green</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="layout_edgesType">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Type</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_edgesType">
        <item>
         <property name="text">
          <string>solid</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>dashed</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="layout_edgesThickness">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Thickness</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_edgesThickness">
        <item>
         <property name="text">
          <string>thin</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>normal</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>thick</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Vertices</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="layout_displayMethod">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Display method</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_displayMethod">
        <item>
         <property name="text">
          <string>none</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>circle</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>square</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="layout_verticesColor">
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Color</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_verticesColor">
        <item>
         <property name="text">
          <string>white</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>black</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>red</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>blue</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>green</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="layout_verticesSize">
      <item>
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Size</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_verticesSize">
        <item>
         <property name="text">
          <string>little</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>pepe</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>big</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
